packages/mask/shared-ui/locales/en-US.json

```
@@ -484,6 +484,7 @@
     "plugin_trader_gas_setting_fast": "Fast",
     "plugin_trader_gas_setting_custom": "Custom",
     "plugin_trader_tx_cost_usd": "(~${{usd}})",
+    "plugin_trader_tx_cost_usd_less": "({{usd}})",
```

I think this is OK to merge with this change.

But I think it would be better like
```suggestion
    "plugin_trader_tx_cost_very_small": "(<$0.01)"
```
```tsx
<span>{ CheckingValueIsLessThan_0_01 ? t('plugin_trader_tx_cost_very_small') : t('plugin_trader_tx_cost_usd', { usd: val})}</span>
```

Might drop `formatUSD` as well.

Could you make this change in another PR?

---

I'm pushing the patch on another branch

---

[developerfred:harmony-network](https://github.com/DimensionDev/Maskbook/pull/6215)